### PR TITLE
Adding Profile Edit View

### PR DESCRIPTION
**Pull request by jesshanta**
Changed the design of profile edit

### CODE REVIEW

**Comment by jpatricia** [comment]
no api call to backend for profile changes? other than that it looks good 👍

File: ChemoTracker/components/Profile/Profile.js [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
-import styles from '../styles/main.js';
-import color from '../styles/color.js';
+import styles from '../../styles/main.js';
+import color from '../../styles/color.js';
 import { StatusBar } from 'react-native';
 
 class Profile extends Component {
+	constructor(props) {
+		super(props);
+		console.log("props of profile", props);
```
Comment (lsivasothy) [inline ChemoTracker/components/Profile/Profile.js]: remove this console log :)

File: ChemoTracker/components/Profile/Cell.js [hunk 2]
```diff
@@ -1,83 +1,14 @@
 import React, { Component } from 'react';
-import { Text, View, Image, TextInput, TouchableWithoutFeedback, TouchableOpacity, Picker } from 'react-native';
+import { Text, View} from 'react-native';
```
Comment (lsivasothy) [inline ChemoTracker/components/Profile/Cell.js]: space is missing after View

File: ChemoTracker/components/Profile/ProfileEdit.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+class ProfileEdit extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      formData: null
+    }
+  }
+
+  dismissView = () => {
+    console.log("dismissView");
```
Comment (lsivasothy) [inline ChemoTracker/components/Profile/ProfileEdit.js]: delete console.log

File: ChemoTracker/components/Profile/ProfileEdit.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    this.props.navigation.goBack();
+  }
+
+  saveChanges = () => {
+    console.log("save changes");
```
Comment (lsivasothy) [inline ChemoTracker/components/Profile/ProfileEdit.js]: delete console.log

File: ChemoTracker/components/Profile/ProfileEdit.js [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    has_accepted_conditions: bool
+    }
+    */
+
+    this.setState({formData:formData})
+    this.props.onFormChange && this.props.onFormChange(formData);
+  }
+
+  handleFormFocus(e, component){
+    //console.log(e, component);
```
Comment (lsivasothy) [inline ChemoTracker/components/Profile/ProfileEdit.js]: delete console.log

File: ChemoTracker/components/Profile/ProfileTop.js [hunk 6]
```diff
@@ -9,6 +9,7 @@ import Icon from 'react-native-vector-icons/FontAwesome';
 class ProfileTop extends Component {
 	constructor(props) {
     super(props);
+		console.log("props of profile top", props);
```
Comment (lsivasothy) [inline ChemoTracker/components/Profile/ProfileTop.js]: delete console.log

File: ChemoTracker/components/Profile/ProfileTop.js [hunk 7]
```diff
@@ -27,11 +28,22 @@ class ProfileTop extends Component {
     }
   };
 
+	openEditView = () => {
+		console.log("open edit view");
```
Comment (lsivasothy) [inline ChemoTracker/components/Profile/ProfileTop.js]: delete console.log

File: ChemoTracker/package.json [hunk 8]
```diff
@@ -34,24 +35,24 @@
...
     "react-native-progressive-input": "^1.0.3",
     "react-native-router-flux": "^4.0.0-beta.21",
+    "react-native-tab-view": "^0.0.74",
     "react-native-tableview-simple": "^0.17.1",
     "react-native-tag-input": "0.0.18",
     "react-native-vector-icons": "^4.5.0",
     "react-navigation": "^1.0.0-beta.11",
-    "react-select": "^1.2.1",
-    "react-native-tab-view": "^0.0.74",
-    "apsl-react-native-button": "^3.0.0"
```
Comment (lsivasothy) [inline ChemoTracker/package.json]: Search uses this! Don't delete.
(make sure all the other features still work when you delete things off here)

File: ChemoTracker/components/Profile/ProfileEdit.js [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+    console.log("dismissView");
+    this.props.navigation.goBack();
+  }
+
+  saveChanges = () => {
+    console.log("save changes");
+    this.props.navigation.goBack();
+  }
+
+  static navigationOptions  = ({ navigation }) => {
```
Comment (lsivasothy) [inline ChemoTracker/components/Profile/ProfileEdit.js]: Maybe move this to ProfileTab? I'm assuming that's where all your navigation logic is?

**Review by lsivasothy — APPROVED** [review]
Other than those minute issues, PR looks good to me!!

File: ChemoTracker/components/Profile/ProfileEdit.js (hunk 9 above)
Comment (jesshanta) [reply ChemoTracker/components/Profile/ProfileEdit.js]: This needs to be here bc the event function for the header should be done here not in Profile tab

**Pull request merged**